backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -1187,6 +1191,17 @@ public void logArtifacts(String experimentRunId, List<Artifact> newArtifacts)
     }
   }
 
+  public void validateMaxArtifactsForTrial(int newArtifactsSize, int existingArtifactsSize)
+      throws ModelDBException {
+    if (app.getTrialEnabled()) {
+      if (existingArtifactsSize + newArtifactsSize > app.getMaxArtifactPerRun()) {
+        throw new ModelDBException(
+            "Maximum " + app.getMaxArtifactPerRun() + " artifacts are allow in the experimentRun",
```

Comment:
"LIMIT_RUN_ARTIFACT_NUMBER : Maximum " + app.getMaxArtifactPerRun() + " artifacts are allow in the experimentRun",
this should look like above

Reply:
Added